src/Sprites/Animation.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "../Point.h"
+#include "../Rect.h"
+#include <vector>
+#include <cstdint>
+
+struct Animation {
+	struct Frame {
+		struct SubFrame {
```

Comment:
I suggest renaming `SubFrame` to `Layer`. That would more closely match terminology in popular graphics packages.

Back in the day, I was never very happy with the original naming of that field. At the time, I didn't feel I had proper terminology to really describe what it was about. Now I'm thinking `Layer` is a much more obvious name.

Reply:
Layer sounds good to me. I'll update shortly. The other variant I saw was FrameComponent.